src/libraries/NetCoreAppLibrary.props

```diff
@@ -149,5 +149,18 @@
   <!-- Make NetCoreAppLibrary available as an item. -->
   <ItemGroup>
     <NetCoreAppLibrary Include="$(NetCoreAppLibrary)" />
+    <NetCoreAppLibrary Update="
```

Comment:
This seems kind of fragile. Can we add validation when building a project that is inbox and set's IsPackable to true, that it contains `IsPackable=true` in this item?

Reply:
This PR already adds protection for that. See the changes in packaging.targets.

Reply:
A cool. Somehow missed it, thanks!